src/catalogue/category/catalogueCard.component.tsx

```diff
@@ -88,9 +90,17 @@ function CatalogueCard(props: CatalogueCardProps) {
             minWidth: 0,
           }}
         >
-          <div>
-            <Typography>{catalogueCategory.name}</Typography>
-          </div>
+          <Grid>
+            <Grid position={'relative'}>
+              <Typography>{catalogueCategory.name}</Typography>
+            </Grid>
+
+            <Grid position={'absolute'} bottom={5} right={12}>
+              <Typography color="text.secondary">
```

Comment:
```suggestion
              <Typography fontSize="0.8rem" color="text.secondary">
```

Can you make the font smaller.